docs/API/event_api.md

```diff
@@ -0,0 +1 @@
+## Event API
```

Comment:
Empty doc?

I think that as least a TBD/WIP mark should be included.

Reply:
This documentation file was just an idea that did not make much sense. Fixed in 45e10846a172668d524e38b689a570f702c07316